app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -102,12 +102,21 @@ <h2>Approved</h2>
       <div class="float-end">
         <button class="btn btn-primary">Send to Registrar</button>
       </div>
-      {% endif %}
+     {% endif %}
       </div>
     </div>
 
 <div id="wrapper">
     <h1 class="text-center pt-1 pb-4">Service-Learning Faculty</h1>
+    <div style="text-align:right; width:100%; padding:0; margin-bottom: -42px; display: inline-block" >
```

Comment:
Change as many of the style tags to their BS5 class equivalents. E.g. p-0 and w-100 are BS classes that should do the same thing. Like above, look at the utilities found here: https://getbootstrap.com/docs/5.0/utilities/spacing/

Reply:
One trouble they had is that we'd have to compile our own bootstrap css with negative margin enabled to use their negative margin classes. And we haven't done that yet